mobile/src/main/java/org/openhab/habdroid/background/ItemUpdateWorker.kt

```diff
@@ -221,6 +225,33 @@ class ItemUpdateWorker(context: Context, params: WorkerParameters) : Worker(cont
         }
     }
 
+    private fun handleVoiceCommand(connection: Connection, value: ValueWithInfo): Result {
+        val headers = mapOf("Accept-Language" to Locale.getDefault().language)
+        val result = try {
+            runBlocking {
+                connection.httpClient
+                    .post("rest/voice/interpreters", value.value, headers = headers)
+                    .asStatus()
+            }
+        } catch (e: HttpClient.HttpException) {
+            if (e.statusCode == 404) {
+                Log.d(TAG, "Voice interpreter endpoint returned 404, falling back to item")
+                runBlocking {
+                    connection.httpClient
+                        .post("rest/items/VoiceCommand", value.value)
+                        .asStatus()
```

Comment:
If that statement throws, the app will crash (unlike before, where there was a nested try-catch).

Reply:
Added try/catch here